agent/azure-monitor-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemExporterTest.java

```diff
@@ -238,7 +239,7 @@ public void dualIkeyBatchWithDelayAndRedirectFlagFalseTest() {
   @Test
   public void initOtelResourceAttributesTest() {
     envVars.set("OTEL_RESOURCE_ATTRIBUTES", "key1=value%201,key2=value2,key3=value%203");
-    Map<String, String> attributes = TelemetryItemExporter.initOtelResourceAttributes();
+    Map<String, String> attributes = initOtelResourceAttributes();
```

Comment:
I prefer to reserve static method imports for well-known methods, e.g. https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports (otherwise it's a bit harder to read, as it's easily confused with a local method)